### PR TITLE
Fix ResNet18 pruning config

### PR DESCRIPTION
**Pull request by vshampor**

### CODE REVIEW

File: examples/classification/configs/pruning/resnet18_pruning_magnitude.json [hunk 1]
```diff
@@ -26,9 +26,9 @@
    "compression": [
        {
             "algorithm": "filter_pruning",
+            "pruning_init": 0.1,
```
Comment (AlexKoff88) [inline examples/classification/configs/pruning/resnet18_pruning_magnitude.json:29]: I didn't get why it should be so.

I have all the cofigs looking as follows:
```
"compression": [
       {
          "algorithm": "filter_pruning",
          "params": {
              "schedule": "exponential",
              "pruning_init": 0.1,
              "pruning_target": 0.4,
              "pruning_steps": 15,
              "weight_importance": "geometric_median" / "L2"
          }
        }
     ]
```

and they work fine!

Comment (asenina) [reply examples/classification/configs/pruning/resnet18_pruning_magnitude.json:29]: @AlexKoff88   In PR #231 we moved `pruning_init` to the logic of the algorithm.

**Comment by vshampor** [comment]
Jenkins please retry a build

**Comment by vshampor** [comment]
Jenkins please retry a build

**Comment by vshampor** [comment]
Fixed as part of #314

**Pull request closed without merging**